artifacts/openapi.yaml

```diff
@@ -612,17 +612,27 @@ components:
           type: string
           pattern: ^[\sa-zA-Z0-9-_()><\[\]]+$
           x-unique: global
+        dhcp_v4interface:
+          description: |-
+            DHCPv4 Client Configuration.
+          $ref: '#/components/schemas/Device.Dhcpv4client'
+          x-field-uid: 9
+        dhcp_v6interface:
+          description: |-
+            DHCPv6 Client Configuration.
+          $ref: '#/components/schemas/Device.Dhcpv6client'
+          x-field-uid: 10
       required:
       - mac
       - name
     Ethernet.Connection:
       description: |-
-        Ethernet interface connection to a port, LAG or VXLAN tunnel.
+        Ethernet interface connection to a port, LAG, VXLAN tunnel or another Chained Ethernet.
```

Comment:
Is "another" word required? Yes, if "port, LAG or VxLAN" are chained ethernet connection. No, if it is not.

Also "Chained" should be in small i.e. "chained"?